dhis-2/dhis-api/src/main/java/org/hisp/dhis/user/sharing/Sharing.java

```diff
@@ -168,7 +154,7 @@ public void addDtoUserGroupAccess( org.hisp.dhis.user.UserGroupAccess userGroupA
 
     public void addUserGroupAccess( UserGroupAccess userGroupAccess )
     {
-        if ( userGroupAccess == null )
```

Comment:
Note this change. I assume this check was meant to initialise the map in case it is null currently but this did check the wrong variable.